bigquery/google/cloud/bigquery/table.py

```diff
@@ -1334,8 +1339,26 @@ def _to_dataframe_tabledata_list(self, dtypes):
         """Use (slower, but free) tabledata.list to construct a DataFrame."""
         column_names = [field.name for field in self.schema]
         frames = []
+
+        # report progress if tqdm installed
+        pbar = None
+        if tqdm is not None:
+            try:
+                pbar = tqdm.tqdm(
+                    desc="Downloading", total=self.total_rows, unit="rows"
+                )
+            except (KeyError, TypeError):
+                # tqdm error
```

Comment:
I'd like this comment to explain a little more why these errors might happen. And more importantly why we are letting them pass (because a broken progress bar shouldn't stop us from downloading results).

Reply:
My thinking here was to protect from something like an interface change in `tqdm`. Indeed the fallback should just be to not show a progress bar.